### DISCUSSION TITLE
How do I change how the date is shown

### DISCUSSION THREAD

**Discussion by TheDoubleH** [HEIGE-PCloud/DoIt#353] [Q&A] [answered]
First off: Great theme - I really love it!

I am based in the US, but my audience will be global. Is there a way to specify a date format like these:

- November 17, 2021
- Nov 17, 2021
- 17. November 2021
- 11/17/21

**Comment by HEIGE-PCloud** [comment, marked as answer]
You should be able to customize the date format by changing the `[params.dateFormat]`.

Here is a [post](https://gohugohq.com/howto/hugo-dateformat/) explaining what kind of format you can put in.

For example

```toml
[params]
  dateFormat = "Monday, Jan 2, 2006"
```

produces

![image](https://user-images.githubusercontent.com/52968553/142278753-f6d78ac9-f1da-43db-9c86-8082d64c406b.png)

Similarly, you can change the date format for the section pages and the list pages through `params.section.dateFormat` and `params.list.dateFormat` as shown in the [sample config](https://hugodoit.pages.dev/theme-documentation-basics/#site-configuration).
